mindsdb/integrations/handlers/langchain_handler/README.md

```diff
@@ -0,0 +1,52 @@
+### Examples of usage
+
+**Create model for chatbot (conversational mode):**
+
+Openai model
+```sql
+CREATE MODEL airline_model
+	PREDICT answer USING
+	engine = "langchain_engine",
+	input_column = "question",
+	openai_api_key = "<open api key>",
+	mode = "conversational",
+	user_column = "question",
+	assistant_column = "answer",
+	model_name = "gpt-3.5-turbo",
+	verbose=True,
+	prompt_template="Answer the user input in a helpful way";
+```
+
+Anyscale provider (defined in 'provider' param)
+```sql
+CREATE MODEL airline_model
+	PREDICT answer 
+	USING
+	engine = "langchain_engine",
+	provider='anyscale',
+	model_name = "mistralai/Mistral-7B-Instruct-v0.1",
+	anyscale_api_key = '<anyscale api key>',
```

Comment:
@ea-rus 
This one should be `anyscale_endpoints_api_key`.

Reply:
provider name is used as part of the key
to use param name `anyscale_endpoints_api_key` we need also to change provider param in the same command:
provider='anyscale_endpoints'

Reply:
Let's change this name to anyscale_endpoints - so that it is consistent with [anyscale_endpoints_handler](https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/integrations/handlers/anyscale_endpoints_handler/anyscale_endpoints_handler.py#L24).

Reply:
@paxcema 
I discussed with @ea-rus offline that we'll keep anyscale_api_key name here because it doesn't use our handler but directly Anyscale API from LangChain.